crates/typst-pdf/src/catalog.rs

```diff
@@ -178,6 +178,23 @@ pub fn write_catalog(
         }
     }
 
+    if !ctx.references.embedded_files.is_empty() {
+        {
+            let mut name_dict = catalog.names();
+            let mut embedded_files = name_dict.embedded_files();
+            let mut names = embedded_files.names();
+            for (name, file_ref) in &ctx.references.embedded_files {
+                names.insert(Str(name.as_bytes()), *file_ref);
+            }
+        }
+
+        // Todo: this is for PDF 2.0; we should probably check the exporting version
+        let mut associated_files = catalog.insert(Name(b"AF")).array().typed();
```

Comment:
My document reader and an online PDF/A validator do not complain about this, but as far as I understand, this is for PDF 2.0 and we are exporting a 1.7 document. Can this stay as it is?

Reply:
I have the same question for `AFRelationship` in the file spec.

Reply:
If it's not 1.7, we should probably remove it. Or is this necessary for embedded files to work?

Reply:
Both come from the ZUGFeRD standard and can probably be removed for "normal" file embeddings. I will try it out.

Reply:
The `AF` and `AFRelationship` keys for PDF 1.7 are described in the normative ISO 19005-3 (PDF/A-3) Annex E. They may be present in a PDF 1.7 file exactly if the file claims PDF/A-3 compliance.

Reply:
Perfect, thank you. Then I will put them behind a check for PDF/A-3.